modules/ROOT/pages/for-each-scope-concept.adoc

```diff
@@ -16,24 +16,37 @@ The For Each scope performs a similar task to using a Splitter and then an Aggre
 
 * By default For Each tries to split the payload. If the payload is a simple Java collection, the For Each scope can split it without any configuration.
 * For Each will not modify the current payload.
-* For non-Java collections, such as XML or JSON, you need to use a DateWeave expression to split data. You use the Collection (`collection`) field for this purpose.
+* For non-Java collections, such as XML or JSON, you need to use a DateWeave expression to split data. You use the *Collection* field for this purpose.
 +
-Here, the Collection field in For Each is set to iterate over an array stored in `payload.topics`.
+Here, the *Collection* field in For Each is set to iterate over an array stored in `payload.topics`:
 +
 image::component-foreach-example.png[For Each Component]
 
-
+Each item in the collection is stored in the payload when processed through the For Each Scope. For example, in the scenario detailed above each split item is stored in `payload.topic` during each iteration.
```

Comment:
this should be just `payload`, not `payload.topic`. Can you check in the example?

Reply:
You are correct, I tested locally and the split items are stored in `payload`. I'll deleted the second sentence here and will update the first one.